docs/guides/themeable-to-emotion-migration-guide.md

```diff
@@ -0,0 +1,459 @@
+
+We feel that it is very important that InstUI is easy to use and learn, so we try to use popular, well maintained and well documented open source solutions whenever possible.
+
+This is why we decided to replace our custom-made theming library (ui-themeable) with a broadly adopted one. The two main candidates were [emotion](https://emotion.sh) and [styled components](https://styled-components.com), we decided to use `emotion`. If you are interested in our detailed reasoning you can read it [here](https://gist.github.com/serikjensen/4ba00b653efac1dbf80543c529adabbc).
+
+- [The emotion package](#themeable-to-emotion-migration-guide/#themeable-to-emotion-migration-guide-for-version-8.0-the-emotion-package)
+- [ApplyTheme to EmotionThemeProvider](#themeable-to-emotion-migration-guide/#applyTheme-to-emotionThemeProvider)
+- [Component Migration steps](#themeable-to-emotion-migration-guide/#component-migration-steps)
+  - [Refactor theme.js](#themeable-to-emotion-migration-guide/#component-migration-steps-refactor-theme.js)
+  - [Create the styles.js file](#themeable-to-emotion-migration-guide/#component-migration-steps-create-the-styles.js-file)
+  - [Make changes in the component](#themeable-to-emotion-migration-guide/#component-migration-steps-make-changes-in-the-component)
+  - [Example Component](#themeable-to-emotion-migration-guide/#component-migration-steps-example-component)
+  - [Theme tests](#themeable-to-emotion-migration-guide/#component-migration-steps-theme-tests)
+
+### The emotion package
+
+The `@instructure/emotion` package contains our implementation of the emotion theming library. You can find the full documentation [here](#emotion).
+
+Just like [**ui-themeable**](#https://instructure.github.io/instructure-ui-legacy-docs/#ui-themeable), **emotion** is used by:
+
+- adding a decorator to our themeable components,
+- mapping the global theme variables to component variables (in the `theme.js` file),
+- applying styles using the component variables (instead of `styles.css` file, now we use a `styles.js` file),
+- and applying themes by wrapping the component in a theme provider component.
+
+```javascript
+---
+embed: true
+---
+<ToggleBlockquote
+  summary="Migrating from ui-themeable"
+>
+  <ToggleBlockquote.Paragraph>
+    The usage is very similar, but migrating will require some refactoring, converting <code>css</code> to <code>js</code> and moving styling logic from the components to the <code>styles.js</code> file.
+  </ToggleBlockquote.Paragraph>
+  <ToggleBlockquote.Paragraph>
+    This guide will explain every step in detail, showing many examples, but <strong>feel free to contact the InstUI team</strong> if you need further help.
+  </ToggleBlockquote.Paragraph>
+</ToggleBlockquote>
+```
+
+## ApplyTheme to EmotionThemeProvider
```

Comment:
We will have a codemod for this, right? If yes we should mention it here

Reply:
We might, but since we don't know for sure what we can and can't solve with codemods, I wanted to leave it out for now. Once we have all the codemods, we can update the docs with their guides.